core_contracts/governance/governance.py

```diff
@@ -43,7 +46,10 @@ def on_install(self) -> None:
     def on_update(self) -> None:
         super().on_update()
         self._time_offset.set(DAY_START + U_SECONDS_DAY * (DAY_ZERO + self._launch_day.get() - 1))
```

Comment:
Let's please remove this line. No need to set this again.

Reply:
Got it. Wasn't sure if it had some purpose still. Removing.